03-transactions.md

```diff
@@ -117,11 +117,59 @@ If a revoked commitment transaction is published, the other party can spend this
 
 #### `to_remote` Output
 
-This output sends funds to the other peer and thus is a simple P2WPKH to `remotepubkey`.
+If `option_anchor_outputs` applies to the commitment transaction, the `to_remote` output is encumbered by a one block csv lock.
+
+    <remote_pubkey> OP_CHECKSIGVERIFY 1 OP_CHECKSEQUENCEVERIFY
+
+The output is spent by a transaction with `nSequence` field set to `1` and witness:
+
+    <remote_sig>
+
+Otherwise, this output is a simple P2WPKH to `remotepubkey`.
+
+#### `to_local_anchor` and `to_remote_anchor` Output (option_anchor_outputs)
+
+This output can be spent by the local and remote nodes respectively to provide incentive to mine the transaction, using child-pays-for-parent. Both
+anchor outputs are always added, except for the case where there are no htlcs and one of the parties has a commitment output that is below the dust limit. 
+In that case only an anchor is added for the commitment output that does materialize. This typically happens if the initiator closes right after opening
+(no `to_remote` output).
+
+    <local_funding_pubkey/remote_funding_pubkey> OP_CHECKSIG OP_IFDUP
+    OP_NOTIF
+        OP_16 OP_CHECKSEQUENCEVERIFY
+    OP_ENDIF
+
+Each party has its own anchor output that locks to their funding key. This is to prevent a malicious peer from attaching child transactions with a low fee
+density to an anchor and thereby blocking the victim from getting the commit tx confirmed in time. This defense is supported by a change in Bitcoin core 0.19:
+https://github.com/bitcoin/bitcoin/pull/15681. This is also the reason that every non-anchor output on the commit tx is CSV locked.
+
+To prevent utxo set pollution, any anchor that remains unspent can be spent by anyone after the commitment tx confirms. This is also the reason to lock
```

Comment:
> can be spent by anyone*

*who knows to grind for matching P2WSH outputs that match the given lightning script and revealed funding pubkey

Reply:
Yes, by anyone with sufficient knowledge of bitcoin. I don't think that needs further clarification in the spec. Will resolve the comment for now, let me know if you think this needs to be revised.